osg-token-renewer-setup.sh

```diff
@@ -9,13 +9,15 @@ usage () {
   echo "   eg: $(basename "$0") myclient123"
   echo
   echo "Options:"
-  echo "  --pw-file /path/to/pwfile"
+  echo "  --pw-file /path/to/pwfile    Specify a password file"
+  echo "  --manual                     Manual client registration"
   exit 1
 }
 
```

Comment:
```suggestion
pwfile=
manual=
```
I like it when things have a default value.